### PR TITLE
Use alluxio fuse sdk script only

### PR DESCRIPTION
**Pull request by LuQQiu**
Remove Alluxio-fuse script. Rename alluxio-fuse-sdk script to alluxio-fuse script.
Modify the generate tarball logics to match the changes

### CODE REVIEW

File: integration/fuse/bin/alluxio-fuse [hunk 1]
```diff
@@ -12,34 +12,46 @@
...
+General options
+\t-h \tprint help
+\t-f \tFUSE foreground option - do not run as daemon
 
-Usage:\talluxio-fuse mount [-n] [-o <mount option>] [-t <sleep seconds>] [mount_point] [alluxio_path]
-\talluxio-fuse mount
+Alluxio mount options
+\t-o <ALLUXIO_PROPERTY_KEY>=<ALLUXIO_PROPERTY_VALUE> \tThe credentials of the target ufs address should be provided here so that alluxio fuse can access the target data set. Other alluxio common/user configuration can also be provided here
+\t-o fuse=<version> (Default=\"3\") \tUnderlying libfuse version to use. Set to 2 if you are using fuse 2.
+\t-o data_cache=<local_cache_directory> (Default=\"\" which means disabled) \tLocal folder to use for local data cache.
```
Comment (ssz1997) [inline integration/fuse/bin/alluxio-fuse:27]: rename to `local_cache_dir`?

File: integration/fuse/bin/alluxio-fuse [hunk 2]
```diff
@@ -50,142 +62,293 @@ get_env() {
...
+# Globals:
+#   None
+# Arguments:
+#   mount_point: Local FUSE mount point
+# Outputs:
+#   0 if fuse mounted, 1 otherwise
+#######################################
+fuse_mounted() {
+  if [[ $# -lt 1 ]]; then
+    err "ufs_address and mount_point should be provided to wait for FUSE to be mounted"
```
Comment (ssz1997) [inline integration/fuse/bin/alluxio-fuse:252]: Error message seems wrong

File: integration/fuse/bin/alluxio-fuse [hunk 3]
```diff
@@ -50,142 +62,293 @@ get_env() {
...
   fi
-  # TODO(lu) check if fuse_pid contains multiple pids
-  local fuse_pid="$(fuse_stat | awk '{print $1,$2}' | grep -w ${mount_point} | awk '{print $1}')"
-  if [[ -z ${fuse_pid} ]]; then
-    umount_fuse
-    return $?
+  declare -r mount_point="$1"
+  fuse_mount_info="$(mount | grep " ${mount_point} ")"
+  if [[ -n "${fuse_mount_info}" ]]; then
+    return 0 # true
```
Comment (ssz1997) [inline integration/fuse/bin/alluxio-fuse:259]: The comment seems unnecessary

File: integration/fuse/bin/alluxio-fuse [hunk 4]
```diff
@@ -50,142 +62,293 @@ get_env() {
...
+  
+  if [[ "${force_kill}" = 'true' ]] ; then
     kill -9 "${fuse_pid}"
     echo "Forcibly killed fuse process ${fuse_pid}"
-    umount_fuse
+    umount_fuse "${mount_point}"
     return $?
   fi
-
+
```
Comment (ssz1997) [inline integration/fuse/bin/alluxio-fuse:313]: extra spaces

File: integration/fuse/bin/alluxio-fuse [hunk 5]
```diff
@@ -199,131 +362,113 @@ umount_fuse() {
...
+    err "At least one command should be passed in"
+    echo -e "${USAGE}"
     exit 1
   fi
-  
+
   get_env
-  if [[ check_fuse_jar == 1 ]]; then
+
+  if ! check_fuse_jar; then
```
Comment (ssz1997) [inline integration/fuse/bin/alluxio-fuse:446]: Double brackets for if

**Review by ssz1997 — APPROVED** [review]
Overall LGTM. Just some minor suggestions

**Review by beinan — APPROVED** [review]
lgtm

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Pull request merged**